build/cloud/01-namespace.yaml

```diff
@@ -2,3 +2,5 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: kubeedge
+  labels:
+    kubeedge: cloudcore
```

Comment:
Why do we need this label? :) And the `kubeedge` ns will be used for all the components for kubeedge cloudside, not only for cloudcore here.

Reply:
That's right I wanted to add the `k8s-app: kubeedge` label. Thanks for catching this one !